neural_compressor/torch/quantization/config.py

```diff
@@ -147,6 +147,7 @@ def __init__(
         quant_lm_head: bool = False,
         # Tuning space
         white_list: Optional[List[OP_NAME_OR_MODULE_TYPE]] = DEFAULT_WHITE_LIST,
+        **kwargs,
```

Comment:
Do we need to pass it to `TorchBaseConfig`?

Reply:
Not need it for now. We are using the `__init__` method from `BaseConfig` . 
If necessary, we can add the `__init__` method to `TorchBaseConfig`, what's your opinion?

Reply:
There's no need to add `__init__` for `TorchBaseConfig` until it has a specific logical difference from `BaseConfig`.

If there is common logic to handle `kwargs`, it would be better to pass it to `BaseConfig`.

Reply:
I agree with you.